Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.19 as build
```

Comment:
`as build` is here because I assume size will matter. This enables [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to be use for such an optimization.